don't use -static-libstdc++, causing problems like "glibc detected *** python: free(): invalid pointer"
cfr. https://github.com/numba/llvmlite/issues/93;
diff --git a/llvmlite-0.11.0/ffi/Makefile.linux.orig b/llvmlite-0.11.0/ffi/Makefile.linux
index 68181f6..b899127 100644
--- a/llvmlite-0.11.0/ffi/Makefile.linux.orig
+++ b/llvmlite-0.11.0/ffi/Makefile.linux
@@ -17,9 +17,7 @@ OUTPUT = libllvmlite.so
 all: $(OUTPUT)
 
 $(OUTPUT): $(SRC) $(INCLUDE)
-	# static-libstdc++ avoids runtime dependencies on a
-	# particular libstdc++ version.
-	$(CXX) -static-libstdc++ -shared $(CXXFLAGS) $(SRC) -o $(OUTPUT) $(LDFLAGS) $(LIBS)
+	$(CXX) -shared $(CXXFLAGS) $(SRC) -o $(OUTPUT) $(LDFLAGS) $(LIBS)
 
 clean:
 	rm -rf test $(OUTPUT)
